python/py-tmuxp/Portfile

```diff
@@ -19,24 +21,34 @@ description         tmux session manager.
 long_description    ${description}
 
 homepage            https://github.com/tmux-python/tmuxp/
```

Comment:
if you keep the github PG then you can remove the `homepage` field as it is set automatically